src/api.ts

```diff
@@ -70,6 +70,20 @@ export interface SpawnTokenRevokeResult {
   status: 'ok',
 }
 
+export type BackendWithStatus = {
+  name: string
+  created_at: string
+  service_name: string
+  cluster_name: string
+  account_name: string
+  status?: string | null
```

Comment:
hmm, when can this be null?

Reply:
Backends have no status if they never get assigned to a drone.

Reply:
IIRC a backend doesn't have a status between the spawn request and the first status message update from the drone.

But can this and `status_timestamp` really be `undefined` OR `null`? We should probably stick to one.

Reply:
What does `serde_json` do with `Option<String>`s?

Reply:
I think it just leaves them out. I'll drop the `null` here.